addons/web/static/src/search/control_panel/control_panel.js

```diff
@@ -93,6 +97,9 @@ export class ControlPanel extends Component {
             "";
         this.currentActiveId = this.env.searchModel?.globalContext.active_id || false;
         this.embeddedActionsKey = `${this.parentActionId}+${this.currentActiveId || ""}`;
+        this.isEmbeddedFetchedKey = `isEmbeddedFetched${this.parentActionId}+${
+            this.currentActiveId || ""
+        }+${user.userId}`;
         this.embeddedActionsConfig = user.settings.embedded_actions_config_ids || {};
```

Comment:
That one is confusing.
If the user object has the item, the current component has a direct reference to a datum inside the user object.
It is error prone because anyone can mistakenly override the internal state of user.
Also, if the new object is taken instead, then that override behavior stops.

What should be done is
- separate the and encapsulate the sources of truth
  eg: the user object (or another service in charge of holding the data) should be the sole owner of the data.
   the control panel merely use that data and copy them as its own state, and calls methods on the original owner to update things.
- at least here we should have a deepCopy of sorts.